node/src/WebRtcTransport.ts

```diff
@@ -12,24 +12,34 @@ import {
 import { WebRtcServer } from './WebRtcServer';
 import { SctpParameters, NumSctpStreams } from './SctpParameters';
 
-export type WebRtcTransportOptions =
+export interface WebRtcTransportListenIndividual
```

Comment:
Why interface? I know it's ok but we use type everywhere else.

Reply:
Hm, there are actually a few other places where interfaces are used, I can change it back if you'd like. Personally though I find it strange to use `type` for anything other than type aliases.

Changed all to `type` for consistency in this file at least.